BoardConfigCommon.mk

```diff
@@ -193,6 +202,13 @@ PRODUCT_SOONG_NAMESPACES += \
 # Thermal
 USE_DEVICE_SPECIFIC_THERMAL := true
 
+# Treble
+PRODUCT_FULL_TREBLE_OVERRIDE := true
+
+# VNDK
+BOARD_VNDK_VERSION := current
+BOARD_VNDK_RUNTIME_DISABLE := true
```

Comment:
May i ask what do this param?

Reply:
defines vndk version

Reply:
disables linker namespace isolation, the 2nd flag

Reply:
because our cam blobs don't like it

Reply:
ofc need to patch blobs for that later though